lib/DB/Schema/Result/CourseUser.pm

```diff
@@ -88,19 +88,6 @@ __PACKAGE__->table('course_user');
 
 __PACKAGE__->load_components('InflateColumn::Serializer', 'Core');
 
-our $VALID_PARAMS = {
-	comment        => q{.*},
-	useMathQuill   => q{[01]},
-	useMathView    => q{[01]},
-	displayMode    => q{\w+},
-	status         => q{\w},
-	lis_source_did => q{\w*},
-	useWirisEditor => q{[01]},
-	showOldAnswers => q{[01]}
-};
-
-our $REQUIRED_PARAMS = {};
-
```

Comment:
This is deleted because it was repeated below as switched to being returned from a subroutine instead of just a hash.